src/CornerPlots.jl

```diff
@@ -0,0 +1,296 @@
+using StatsBase
+using CairoMakie
+
+export create_corner_plot 
+
+"""
+    mutable struct PlottingProps
+
+PlottingProps contains the props, units, ranges, and names (latex encouraged)
+for each property to plot.
+"""
+
+@kwdef mutable struct PlottingProps
+    props::Vector{Symbol}
+    units::Vector{Float64}
+    ranges::Vector{Any}    # RTW TODO
+    names::Vector{AbstractString}
+end
+
+function createPlottingProps(props_matrix::Vector{Vector{Any}})
```

Comment:
This is a constructor, usually the notation is to define the constructor with the same name as the struct

Reply:
Also, not sue I understand why you have this extra layer. Any struct you create will have by default a constructor that accepts the individual arguments (as you are using within this constructor). Why don't we simply use that constructor at the user-level?

Reply:
The point of the function is to allow the user to assign some of the priors (or plotting props in this case), while leaving some of the others as defaults. The struct's complain when you don't supply all the arguments.

Reply:
Defaults can be defined in the struct:

```julia
@kwdef struct Point
    a = 1
    b = 2
end

##
Point(a=1, b=4)

##
Point(a=3)

##
Point(b=2)
```